profiler/src/Demos/Shared/Util/Datadog.Demos.Util.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <Platforms>AnyCPU;x64;x86</Platforms>
```

Comment:
nit: I know it was me, who introduced this change, but I'm not sure if it required anymore.

Reply:
I'll look into it. Leave it for now and removing it in https://github.com/DataDog/dd-trace-dotnet/pull/3737